src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -83,6 +83,8 @@ impl LocationState {
         self.permission
     }
 
+    // Performs an access on this index and updates node,
+    // perm and wildcard_state to reflect the transition.
```

Comment:
```suggestion
    /// Performs an access on this index and updates node,
    /// perm and wildcard_state to reflect the transition.
```
To make it actually a doc comment